src/CommunityToolkit.Maui.Core/Views/DrawingView/IDrawingView.shared.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System.Collections.ObjectModel;
+using System.Windows.Input;
+
+namespace CommunityToolkit.Maui.Core.Views;
+
+/// <summary>
+/// The DrawingView allows you to draw one or multiple lines on a canvas
+/// </summary>
+public interface IDrawingView : IView
+{
+	/// <summary>
+	/// Event occurred when drawing line completed
+	/// </summary>
+	event EventHandler<DrawingLineCompletedEventArgs> DrawingLineCompleted;
```

Comment:
So... .NET MAUI doesn't implement `events` on interfaces, to no influence the implementors, so I would say that we should do the same... Instead of an `Event`, you can implement a method, just like the button does as you can see [here](https://github.com/dotnet/maui/blob/main/src/Core/src/Core/IButton.cs)

Reply:
that's interesting. 
I do not see the implementation of these methods in Microsoft.Maui.Controls.Button, although it implements IButton.

Reply:
So, if you see all Controls inside controls are `partial` classes, one of those classes have the BP (basically copied and pasted form Forms) and the new part is [here](https://github.com/dotnet/maui/tree/main/src/Controls/src/Core/HandlerImpl) and the [Button is here](https://github.com/dotnet/maui/blob/main/src/Controls/src/Core/HandlerImpl/Button/Button.Impl.cs)

Reply:
thank you!